src/platforms/native/configuration/transports.mdx

```diff
@@ -45,3 +47,17 @@ int main(void) {
 The transport is invoked in the same thread as the call to
 `sentry_capture_event`. Consider to offload network communication to a
 background thread or thread pool to avoid blocking execution.
+
+## Using an HTTP proxy
+
+The Native SDK allows the configuration of an HTTP proxy through which requests can be  tunneled to our backend. This proxy must allow `CONNECT` requests to establish a proxy connection. It can be configured via the following option interface:
+
+```c
+sentry_options_t *options = sentry_options_new();
+sentry_options_set_http_proxy(options, "http://my.proxy:10010");
+sentry_init(options);
+
+/* ... */
+```
+
+Both transport implementations (`Curl` and `WinHTTP`) honor the configuration on all supported platforms.
```

Comment:
though we don’t forward it to the crashpad, do we?

Reply:
Exactly and I looked into the transport implementations of crashpad, and they explicitly all do not proxy. That should probably be an alert-box in this doc. And also something we should rectify.

Reply:
Another thing I thought would make sense would be to honor the `http_proxy` (etc.) environment variable(s) at some point. I also saw that some other SDKs already do that when I reviewed the basic options sections.